example/src/tests/groupTests.ts

```diff
@@ -1993,6 +1994,74 @@ test('can list groups does not fork', async () => {
   return true
 })
 
+test('can create new installation without breaking group', async () => {
+  const keyBytes = new Uint8Array([
+    233, 120, 198, 96, 154, 65, 132, 17, 132, 96, 250, 40, 103, 35, 125, 64,
+    166, 83, 208, 224, 254, 44, 205, 227, 175, 49, 234, 129, 74, 252, 135, 145,
+  ])
+  const wallet1 = new Wallet(
+    '0xc54c62dd3ad018ef94f20f0722cae33919e65270ad74f2d1794291088800f788'
+  )
+  const wallet2 = new Wallet(
+    '0x8d40c1c40473975cc6bbdc0465e70cc2e98f45f3c3474ca9b809caa9c4f53c0b'
+  )
+  const client1 = await Client.create(wallet1, {
+    env: 'local',
+    appVersion: 'Testing/0.0.0',
+    enableV3: true,
+    dbEncryptionKey: keyBytes,
+  })
+  const client2 = await Client.create(wallet2, {
+    env: 'local',
+    appVersion: 'Testing/0.0.0',
+    enableV3: true,
+    dbEncryptionKey: keyBytes,
+  })
+
+  const group = await client1.conversations.newGroup([wallet2.address])
+
+  await client1.conversations.syncGroups()
+  await client2.conversations.syncGroups()
+
+  const client1Group = (await client1.conversations.listGroups()).find(
+    (g) => g.id === group.id
+  )
+  const client2Group = (await client2.conversations.listGroups()).find(
+    (g) => g.id === group.id
+  )
```

Comment:
Total nit but you can actually call just `find_group` directly in V3 instead of iterating over the list.